### ISSUE TITLE
on my own dataset

### ISSUE THREAD

**Issue by chj1933** [YantaoShen/kpm_rw_person_reid#2]
@YantaoShen ,Hello, If I want to use my own dataset, Start to run from scratch, How should I set parameters? Not only fine tune.

**Comment by YantaoShen** [comment]
Hi, I not aware of your dataset, so here I just list my experience of training model on market1501. If you want to train from ImageNet pretrained model, you may try learning rate of 0.01 firstly. weight decay and momentum can be used as the default setting. For batch size, you may try 64 or 128.
